fabtests/pytest/common.py

```diff
@@ -6,11 +6,14 @@ def has_cuda(ip):
     proc = run(["ssh", ip, "nvidia-smi", "-L"])
     return proc.returncode == 0
 
-def has_hmem_support(cmdline_args, host_type, ip):
+def has_hmem_support(cmdline_args):
     from subprocess import run
-    cmd = cmdline_args.populate_command("check_hmem " + cmdline_args.provider, host_type, 10)
-    proc = run(["ssh", ip, cmd])
-    return proc.returncode == 0
+    test = UnitTest(cmdline_args, "check_hmem")
```

Comment:
I am not sure about using `UnitTest` for this purpose. Maybe introduce another option to `populate_command`?

Reply:
Since this program is very special and may not accept more opts than `-p`, I am going to construct the command manually rather than using `populate_command`.